### PR TITLE
github: add templates for bug reports and pull requests

### LINKED ISSUE #1079: Create Google Group for Users questions
Originally we had a mailing list for user questions, but that required some maintenance and received lots of spam, so we decided to close it on February.

We have decided to create a google groups so users can share their experiences there and leave Github issues for bug reports and project scheduling.

We don't plan to provide support there during our work hours, but anyone can provide feedback or help.
 
https://groups.google.com/forum/#!forum/ivozprovider-users

 We have to:

- [x] Edit main README contact section
- [x] Add entry in FAQ with the group link 
- [x] Add issue template to redirect user questions to the group
- [ ] Update minimal group configuration (welcome message, logo and so on)
- [ ] ...

### PR DESCRIPTION
**Pull request by Kaian**
Also include all ivozprovider-users group in documentation and
updates FAQ to a numeric list for easy reference.

Fixes  #1079

### CODE REVIEW

**Comment by Kaian** [comment]
You can check the templates in https://github.com/kaian/ivozprovider project

File: FAQ.md [hunk 1]
```diff
@@ -46,10 +46,10 @@ That address must be present in your network interfaces in order to bind service
 ### I have trouble with my environment!
 
 We don't provide any kind of free support for installations. If you have trouble, you can contact our sales department
-at comercial@irontec.com
+at comercial@irontec.com or ask in [ivozprovider-users](https://groups.google.com/forum/#!forum/ivozprovider-users) group.
 
-If by any chance you receive support in Github or #ivozprovider freenode IRC channel, it's most probably provided
-during developers spare time, so be thankful for the people giving part of their time.
+If by any chance you receive support in the grup or #ivozprovider freenode IRC channel by any developer, it's most
```
Comment (Kaian) [inline FAQ.md]: the grup :(

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### Description
+<!--
+Explain what you did, what you expected to happen, and what actually happened.
+-->
+
+---
+
+- IvozProvider version: <!-- You can get it from dpkg -l ivozprovider -->
+
+- Sections related: <!-- Mark X between brackets to one or more sections -->
```
Comment (cruzccl) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Related sections

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+### Description
+<!--
+Explain in detail what the feature is about.
+-->
+
+---
+
+- I'm willing to: <!-- Mark X between brackets if they apply -->
+
+ - [ ] Implemnt this feature myself
```
Comment (cruzccl) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Implement

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<!--
+  - All pull requests must be done against bleeding branch.
+  - All provided code must be GPLv3 license compatible.
+-->
+
+#### Type Of Change <!-- Mark one with X -->
+- [ ] Small bug fix
+- [ ] New feature or enhacement
```
Comment (cruzccl) [inline .github/PULL_REQUEST_TEMPLATE.md]: enhaNcement

**Pull request merged**